contents/articles/2017-01_saliency-in-vr/index.md

```diff
@@ -0,0 +1,21 @@
+--- 
+title: "Saliency in VR: How do people explore virtual environment"
+date: Mon Jan 09 2017 11:45:18 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Vincent et al., proposes new methods to learn and predict time-dependent saliency in VR to make eye tracking expendable..."
```

Comment:
Vincent Sitzmann et al., use Pupil Labs Oculus DK2 add-on to research and propose new methods to learn and predict time-dependent saliency in VR